src/features/navigation/TabBar/routes.ts

```diff
@@ -26,7 +26,7 @@ export const routes: Array<TabRoute> = [
     name: 'Search',
     component: Search,
     pathConfig: {
-      path: 'search',
+      path: 'rechercher',
```

Comment:
Tu as mis `recherche` dans `routing` : 
```suggestion
      path: 'recherche',
```